ndm-operator.yaml

```diff
@@ -139,6 +139,8 @@ spec:
         - name: procmount
           mountPath: /host/proc
           readOnly: true
+        - name: basepath
+          mountPath: /var/openebs
         - name: sparsepath
```

Comment:
is this going to impact the path of the file on blockDevice CR?

Reply:
Reverting this change.